apps/chrome-devtools/src/components/app-connection/app-connection.component.html

```diff
@@ -4,7 +4,7 @@
   </div>
 
   <div *ngSwitchCase="'timeout'" class="alert alert-danger m-2" role="alert">
-    The application does not seem to be compatible with the Otter Chrome DevTools.
+    The application does not seem to be compatible with the Otter Chrome DevTools or the Otter Chrome Extesion does not have permission to access this page.
```

Comment:
Instead of adding a message, maybe you can just emit and display the error when then `onConnect` 's `chrome.scripting.executeScript` throw

Reply:
Are you saying that there's a way to detect whether  the user did not give permission to access the page or the application is not Otter compatible?

Reply:
When the permission is not given, the call to the Chrome Api is failing (according to the doc).
According to the case descriotion, the missing permission seems to be the page writring permission allowing `chrome.scripting.executeScript` so my question is: why nnot handling this function call failure instead of creating a message containing another possible failing case (that we can actually detect)